### PR TITLE
:bug: Fixing bug with repair allocation

### PR DESCRIPTION
**Pull request by moldis**

### CODE REVIEW

File: cmd/delete.go [hunk 1]
```diff
@@ -33,6 +33,10 @@ var deleteCmd = &cobra.Command{
 			os.Exit(1)
 		}
 		remotepath := cmd.Flag("remotepath").Value.String()
+		blobberUrl := cmd.Flag("blobber_url").Value.String()
+		if len(blobberUrl) > 0 {
+			println("DEVELOPER ONLY. Make sure you understand meaning of blobber_url")
```
Comment (stewartie4) [inline cmd/delete.go:38]: should this be removed from the customer facing CLI?

File: cmd/delete.go [hunk 2]
```diff
@@ -86,6 +94,7 @@ func init() {
 	deleteCmd.PersistentFlags().String("allocation", "", "Allocation ID")
 	deleteCmd.PersistentFlags().String("remotepath", "", "Remote path of the object to delete")
 	deleteCmd.Flags().Bool("commit", false, "pass this option to commit the metadata transaction")
+	deleteCmd.Flags().String("blobber_url", "", "DEVELOPER ONLY, set it if you want delete file from specific blobber")
```
Comment (stewartie4) [inline cmd/delete.go:97]: this can't be merged as part of the customer facing CLI

**Comment by stewartie4** [comment]
What's the high level purpose of this PR?

File: cmd/delete.go (hunk 1 above)
Comment (moldis) [reply cmd/delete.go:38]: @stewartie4 I need this for system_tests to emulate broken files hashes in blobbers

Comment (stewartie4) [reply cmd/delete.go:38]: we shouldn't be modifying the public CLI for testing purposes - customers could misuse this despite the warning 
You can call the blobber REST API directly in system tests to break files on individual blobbers

**Review by vincenzopalazzo — APPROVED** [review]
LGTM

**Comment by stewartie4** [comment]
How do these affect repair system tests @moldis ?

**Review by stewartie4 — CHANGES_REQUESTED** [review]
Remove the developer flags from the user facing CLI
Unless we intend that customers be able to do this?
https://github.com/0chain/zboxcli/pull/135/files#r766613602

**Pull request closed without merging**